server/src/data/flags.ts

```diff
@@ -2,8 +2,8 @@ import { devConfig } from "../config/DevSettings";
 
 export const flags = {
   // Platform
-  viximo: 0,
-  kongregate: 1,
+  viximo: 1,
```

Comment:
Is there any reasons for this change? Can we revert this?

Reply:
I think set this because there's a checking on client if the platform is viximo and it will load the pictures properly

https://github.com/bym-refitted/backyard-monsters-refitted/blob/d08ffdb72e4afda14ee69944a17ea05bd8b06944/client/scripts/com/monsters/maproom_advanced/PopupInfoEnemy.as#L497